setup.py

```diff
@@ -69,7 +73,7 @@
     extras_require={
         "docs": doc_reqs,
         "test": test_reqs,
-        "allen_brain": ["allensdk>=2.13.5"],
+        #"allen_brain": ["allensdk>=2.13.5"],
```

Comment:
I think we can just get rid of AllenSDK thanks to https://github.com/brainglobe/morphapi/pull/69, we just have release a new version of MorphAPI and then update the min version here.